src/Acuminator/Acuminator.Analyzers/StaticAnalysis/LegacyBqlField/LegacyBqlFieldAnalyzer.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.Linq;
+using Acuminator.Utilities.Roslyn.Semantic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Acuminator.Analyzers.StaticAnalysis.LegacyBqlField
+{
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	public class LegacyBqlFieldAnalyzer : PXDiagnosticAnalyzer
+	{
+		public const string CorrespondingPropertyType = "CorrespondingPropertyType";
+		public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(Descriptors.PX1060_LegacyBqlField);
+
+		protected override bool ShouldAnalyze(PXContext pxContext) => pxContext.IsAcumatica2019R1;
+
+		internal override void AnalyzeCompilation(CompilationStartAnalysisContext compilationStartContext, PXContext pxContext)
+		{
+			compilationStartContext.RegisterSymbolAction(
+				c => Analyze(c, pxContext),
+				SymbolKind.NamedType);
+		}
+
+		private async void Analyze(SymbolAnalysisContext context, PXContext pxContext)
```

Comment:
async is not needed here:
```suggestion
		private void Analyze(SymbolAnalysisContext context, PXContext pxContext)
```